E2E-ORS-Sample-For-Java/src/main/java/ObjectReplicationMonitor.java

```diff
@@ -0,0 +1,297 @@
+import com.azure.storage.blob.models.ObjectReplicationPolicy;
+import com.azure.storage.blob.models.ObjectReplicationRule;
+import com.azure.storage.blob.models.AccessTier;
+import com.azure.storage.blob.models.BlobStorageException;
+import java.util.Properties;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.io.InputStream;
+import java.io.IOException;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.File;
+import java.io.ByteArrayInputStream;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+
+public class ObjectReplicationMonitor {
+    /*
+    This class is to demonstrate what object replication does. please read the README in its entirety
+    before running this program and make sure all prerequisites are met.
+     */
+
+    public static void main(String[] args) throws IOException {
+
+        // Get path to config file
+        Path currentPath = Paths.get(System.getProperty("user.dir"));
+        Path pathToDir = Paths.get(currentPath.toString(),
+                "src", "main", "java");
+        String pathToConfig = Paths.get(pathToDir.toString(), "app.config").toString();
+
+        // Get values from config files
+        InputStream input = new FileInputStream(pathToConfig);
+        Properties prop = new Properties();
+        prop.load(input);
+
+        String sourceContainerName = prop.getProperty("sourceContainerName");
+        String sourceStorageAccountConnectionString = prop.getProperty("sourceStorageAccountConnectionString");
+        String destinationContainerName = prop.getProperty("destinationContainerName");
+        String destinationStorageAccountConnectionString = prop.getProperty("destinationStorageAccountConnectionString");
+        String blobsToReplicatePrefix = prop.getProperty("blobsToReplicatePrefix");
+        String archiveMethod = prop.getProperty("archiveMethod");
+
+        // Creating a sample list of blobs to upload and replicate
+        String[] replicatedBlobList = new String[250];
+        int num = 0;
+        for (int i = 0; i < replicatedBlobList.length; i++) {
+
+            replicatedBlobList[i] = blobsToReplicatePrefix + num;
+            num++;
+
+        }
+
+        // Call method to setup blob in source container
+        setupSource(sourceStorageAccountConnectionString, sourceContainerName, replicatedBlobList);
+
+        // Call method to check the replication status of the blob in source container
+        System.out.println("\nOnce the replication process begins, the replication status will output...");
+        List<String> completedEventsList = checkReplication(sourceStorageAccountConnectionString, sourceContainerName, replicatedBlobList);
+
+        // Call method to check the contents of a replicated blob
+        checkContents(sourceStorageAccountConnectionString, sourceContainerName, destinationStorageAccountConnectionString, destinationContainerName, completedEventsList);
```

Comment:
I usually like to keep each line within an 80 char limit, so it's easier to read without having to scroll to the right